jwql/website/apps/jwql/templates/view_image.html

```diff
@@ -34,18 +34,18 @@ <h3>{{ file_root }}</h3>
 		</form>
 		<br>
 
-		<!-- View Header and proposal information -->
+		<!-- Buttons for viewing header, downloading files -->
     	<p>
-	    	<a id="view_header" class="btn btn-primary mx-2" role="button" href="">View Header</a>
-	    	<a class="btn btn-primary mx-2 disabled" role="button" href="#"> View Proposal {{ file_root[2:7] }} in APT</a>
+	    	<a id="view_header" class="btn btn-primary mx-2" role="button" href="{{ url('jwql:view_header', args=[inst, file_root]) }}">View Header</a>
+            <!-- <a id="download_fits" class="btn btn-primary my-2 mx-2" role="button" href='{{ static("") }}preview_images/{{ file_root[:7] }}/{{ file_root }}' download>Download FITS</a>
```

Comment:
@bourque Did we want to implement this again or are we going to hold off on the download feature for now?

Reply:
Yeah I think we should do this in a separate PR.  Re-implementing this is not going to be trivial 😓